3-WebApp-multi-APIs/WebApp-OpenIDConnect-DotNet.csproj

```diff
@@ -18,9 +18,10 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.6.0" />
+    <PackageReference Include="Microsoft.Identity.Client" Version="4.19.0" />
     <PackageReference Include="Microsoft.Identity.Web" Version="0.4.0-preview" />
```

Comment:
you'll have a merge conflict here. Microsoft.Identity.Web is 1.0.0 now

Reply:
Updated to latest GA package.